mvs_eland_tool/local_deploy.py

```diff
@@ -137,7 +137,9 @@ def main(**kwargs):
 
     logging.debug("Accessing script: F0_outputs")
     output_processing.evaluate_dict(
-        dict_values, path_pdf_report=user_input.get("path_pdf_report", None)
+        dict_values,
+        path_pdf_report=user_input.get("path_pdf_report", None),
```

Comment:
Why is "path_pdf_report" a string and not a constant variable?

Reply:
Same reasons as above